src/components/RadioButtonFieldRow.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from "react";
+import {
+  StyleProp,
+  ViewStyle,
+  TextStyle,
+  StyleSheet,
+  TouchableOpacity,
+} from "react-native";
+import { withTheme } from "../core/theming";
+import RadioButton from "./RadioButton";
+import themeT from "../styles/DefaultTheme";
+import { View } from "react-native";
+import Text from "./Text";
+
+interface Props {
+  label: string;
+  backgroundColor?: string;
+  labelColor?: string;
+  radioButtonColor?: string;
+  style?: StyleProp<ViewStyle>;
+  labelStyle?: StyleProp<TextStyle>;
+  onPress?: () => void;
+  selected: boolean;
+  theme: typeof themeT;
+}
+
+const RadioButtonFieldRow: React.FC<Props> = ({
+  label,
+  theme: { colors },
+  backgroundColor = colors.background,
```

Comment:
oh that's a neat trick! does this really work?

Reply:
yes